templates/swift-library/ios/{%= project.name %}ViewManager.m

```diff
@@ -0,0 +1,7 @@
+#import "React/RCTViewManager.h"
```

Comment:
Shoudn't this be `#import <React/RCTViewManager.h>`?

Reply:
I think in this case the Header search paths are the same, so both "global" and "local" import work here